build.gradle

```diff
@@ -157,7 +160,9 @@ dependencies {
     compile "org.json:json:${jsonVer}"
     compile "com.google.inject:guice:${guiceVer}"
     compile "com.google.inject.extensions:guice-assistedinject:${guiceVer}"
-    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:2.23.2"
+    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:${jerseyContainerVer}"
+    compile "org.eclipse.jetty:jetty-servlet:${jettyServletVer}"
+    compile "com.owlike:genson:${gensonVer}"
```

Comment:
Why use this vs `Gson` or `Jackson` ? What advantages does Genson offers ? 

My recommendation is to go with projects that are more stable and have good communities behind them.